prow/sidecar/run.go

```diff
@@ -203,7 +203,12 @@ func (o Options) preUpload() {
 
 	if o.CensoringOptions != nil {
 		if err := o.censor(); err != nil {
-			logrus.Warnf("Failed to censor data: %v", err)
+			log := logrus.WithError(err)
+			handler := log.Warnf
+			if o.CensoringOptions.Strict {
+				handler = log.Fatalf
+			}
+			handler("Failed to censor data: %v", err)
```

Comment:
Would it be possible to upload some kind of tombstone artifact so that this is at least communicated to the end user that comes searching their artifact? A simple .txt file with a simple message would probably save people time and effort to investigate.

Otherwise LGTM